applications/nrf5340_audio/README.rst

```diff
@@ -1473,4 +1466,4 @@ Legal notices for the nRF5340 Audio DK
 
 .. |net_core_hex_note| replace:: The network core for both gateway and headsets is programmed with the precompiled Bluetooth Low Energy Controller binary file :file:`ble5-ctr-rpmsg_<XYZ>.hex`, where *<XYZ>* corresponds to the controller version, for example :file:`ble5-ctr-rpmsg_3216.hex`.
    This file includes the LE Audio Controller Subsystem for nRF53 and is provided in the :file:`applications/nrf5340_audio/bin` directory.
-   If :ref:`DFU <nrf53_audio_app_configuration_configure_fota>` is enabled, the subsystem's binary file will include variables from the naming patterns for FOTA build files (:file:`dev_<Role>_build_<Application version>_ble5-ctr_CPUNET.hex`).
+   If :ref:`DFU <nrf53_audio_app_configuration_configure_fota>` is enabled, the subsystem's binary file will be generate in :file:`build/zephyr/` folder (:file:`net_core_app_signed.hex`).
```

Comment:
```suggestion
   If :ref:`DFU is enabled <nrf53_audio_app_configuration_configure_fota>`, the subsystem's binary file will be generated in the :file:`build/zephyr/` directory and will be called :file:`net_core_app_signed.hex`.
```

Reply:
@tejlmand , net_core_app_signed.hex is not listed as one of the output files here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_build_system.html#output-build-files 

Should we add it to one of the tables?